heliopy/data/util.py

```diff
@@ -210,7 +204,7 @@ def units_attach(data, units):
     return timeseries_data
 
 
-def cdf_units(cdf_, keys=None, manual_units=None):
+def cdf_units(cdf_, manual_units=None):
```

Comment:
@abotiamnot can you check that the changes in this function are okay?

Reply:
Yep! Just gave it a go, works perfectly!